gopro

```diff
@@ -305,6 +358,14 @@ function start_gopro_webcam_mode {
     fi
 
     green "Sucessfully started the GoPro Webcam mode. (The icon on the Camera should have changed)"
+    
+    # Setting Field Of View (FOV)
+    response=$(curl -s ${GOPRO_INTERFACE_IP}${FOV_PATH}${FOV})
+    if [ $? -ne 0 ]; then
+    	yellow "Do not possible to set FOV for gopro"
```

Comment:
This isn't grammatically sound sentence. Maybe it could just be "Failed to set the GoPro FOV"? Then it would match the success case.

Reply:
sorry, I am only interest in test information from reddit linkd with gopro web commands.